### ISSUE TITLE
Question in VQAv2 finetuning in Paper

### ISSUE THREAD

**Issue by sarath-mutnuru** [OFA-Sys/OFA#250]
Hello, thanks for the solid work and making it open source.

I have a question from the paper. In the Appendix A.3 - Details of Downstream Tasks, for Visual Question Answering. It is mentioned that pre-trained OFA is fine-tuned on VQAv2. 
However, I understand that VQAv2 data (along with GQA and VG-QA) is used for pre-training as well.
Does this mean we pre-trained and fine-tuned on same data? If so, how does this make sense conceptually and contribute to increased accuracy?
Thank you.
